doc/source/whatsnew/v1.1.0.rst

```diff
@@ -346,9 +346,8 @@ Plotting
 ^^^^^^^^
 
 - :func:`.plot` for line/bar now accepts color by dictonary (:issue:`8193`).
--
 - Bug in :meth:`DataFrame.boxplot` and :meth:`DataFrame.plot.boxplot` lost color attributes of ``medianprops``, ``whiskerprops``, ``capprops`` and ``medianprops`` (:issue:`30346`)
-
+- :func:`.autocorrelation_plot` and :func:`.lag_plot` now accept ``figsize`` attribute (:issue:`31650`)
```

Comment:
could you check how funcs are represented? i doubt if the dot you put in front of func name is correct.